conftest.py

```diff
@@ -7,7 +7,7 @@
 
 from topo_processor.stac.iter_errors_validator import IterErrorsValidator
 
-pystac.validation.set_validator(IterErrorsValidator())
+# pystac.validation.set_validator(IterErrorsValidator())
```

Comment:
Why is this commented?

Reply:
It was for testing, I'm just updating it.